### PR TITLE
*: enable v3 by default

### PR DESCRIPTION
**Pull request by xiang90**
WOW... We have v3 by default now.

### CODE REVIEW

File: V3DemoProcfile [hunk 1]
```diff
@@ -1,7 +0,0 @@
...
-etcd2: bin/etcd --experimental-v3demo=true --name infra2 --listen-client-urls http://127.0.0.1:22379 --advertise-client-urls http://127.0.0.1:22379 --listen-peer-urls http://127.0.0.1:22380 --initial-advertise-peer-urls http://127.0.0.1:22380 --initial-cluster-token etcd-cluster-1 --initial-cluster 'infra1=http://127.0.0.1:12380,infra2=http://127.0.0.1:22380,infra3=http://127.0.0.1:32380' --initial-cluster-state new --enable-pprof
-etcd3: bin/etcd --experimental-v3demo=true --name infra3 --listen-client-urls http://127.0.0.1:32379 --advertise-client-urls http://127.0.0.1:32379 --listen-peer-urls http://127.0.0.1:32380 --initial-advertise-peer-urls http://127.0.0.1:32380 --initial-cluster-token etcd-cluster-1 --initial-cluster 'infra1=http://127.0.0.1:12380,infra2=http://127.0.0.1:22380,infra3=http://127.0.0.1:32380' --initial-cluster-state new --enable-pprof
-# in future, use proxy to listen on 2379
-#proxy: bin/etcd --name infra-proxy1 --proxy=on --listen-client-urls http://127.0.0.1:2378 --initial-cluster 'infra1=http://127.0.0.1:12380,infra2=http://127.0.0.1:22380,infra3=http://127.0.0.1:32380' --enable-pprof
```
Comment (heyitsanthony) [inline V3DemoProcfile]: I think this still has to stick around until v3 proxy works. Otherwise v3 tools won't work by default with a goreman cluster.

Comment (xiang90) [inline V3DemoProcfile]: sure.

**Comment by heyitsanthony** [comment]
lgtm so long as there's still a v3 goreman file

**Comment by gyuho** [comment]
lgtm.

**Pull request merged**